engine/src/main/java/org/terasology/logic/debug/MovementDebugCommands.java

```diff
@@ -233,22 +236,48 @@ public String stepHeight(@Sender EntityRef client, @CommandParam("height") float
         return "";
     }
 
+    private float getJumpSpeed(float amount){
+        double d = (double)amount;
+        return (float) pow(d,0.6) * 9.36f ;
+    }
+    
+    private float getInteractionRange(float amount){
+        double d = (double)amount;
+        return (float) pow(d,0.6) * 3.9f ;
+    }
+
+    private float getRunFactor(float amount){
+       return 0.15f + (0.64f * amount) - (0.006f * amount * amount);
+    }
+
     @Command(shortDescription = "Sets the height of the player", runOnServer = true,
             requiredPermission = PermissionManager.CHEAT_PERMISSION)
     public String playerHeight(@Sender EntityRef client, @CommandParam("height") float amount) {
```

Comment:
Height has a limitation of `@Range(min = 0, max = 5)` in `CharacterMovementComponent`, that should probably also be checked here...